benchopt/util.py

```diff
@@ -274,7 +274,9 @@ def filter_solvers(solvers, solver_names=None, forced_solvers=None,
     if len(solver_names) > 0:
         solvers = [s for s in solvers
                    if s.name.lower() in solver_names + forced_solvers]
-
+        not_found = list(set(solver_names + forced_solvers) - set(solvers))
+        if not_found:
+            warnings.warn('Solver(s) not found: %s' % ', '.join(not_found))
```

Comment:
Arf sorry wrong button..

I was saying, an error is better as the user requested the solver explicitly.

Also, could you add a test for this? This will be easier with an error.